roles/pulp-webserver/tasks/main.yml

```diff
@@ -13,6 +13,38 @@
   tags:
     - always
 
+- name: Create the /etc/ssl/private dir
+  file:
+    path: /etc/ssl/private
+    state: directory
+    mode: 0755
+
+- name: Create the /etc/ssl/csr dir
+  file:
+    path: /etc/ssl/csr
+    state: directory
+    mode: 0755
+
+- name: Generate an OpenSSL private key
+  openssl_privatekey:
+    path: /etc/ssl/private/pulp.pem
+
+- name: Generate an OpenSSL Certificate Signing Request
+  openssl_csr:
+    path: /etc/ssl/csr/pulp.csr
+    privatekey_path: /etc/ssl/private/pulp.pem
+    country_name: US
+    organization_name: Pulp
+    email_address: example@example.com
+    common_name: "{{ ansible_nodename }}"
```

Comment:
Don't we want to ensure that this is an FQDN? If so, `{{ ansible_fqdn }}` ?

I see we are already using `{{ ansible_nodename }}` in our jinja2 templates, but[ we set content_origin](https://github.com/pulp/ansible-pulp/blob/master/example-use/group_vars/all) to `{{ ansible_fqdn }}`./